std-bits/bench-processor/src/main/java/org/enso/benchmarks/processor/GenerateBenchSources.java

```diff
@@ -0,0 +1,15 @@
+package org.enso.benchmarks.processor;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Just a dummy annotation to force the {@link BenchProcessor} to generate JMH sources
+ * for all the discovered Enso benchmarks.
+ */
+@Retention(RetentionPolicy.CLASS)
+@Target(ElementType.TYPE)
+public @interface GenerateBenchSources {
```

Comment:
I'd add two arguments to this annotation:
- location of `.enso` file to evaluate
- name of a symbol to invoke to obtain all `BenchSuite`

Reply:
In the most recent version of this PR - https://github.com/enso-org/enso/blob/f5622840c9ec7e3a4f978a57898f21b238f88945/lib/scala/bench-processor/src/main/java/org/enso/benchmarks/processor/GenerateBenchSources.java, there are three arguments:
- `projectRootPath` that is a relative path to the project root
- `moduleName` that is FQN of the module
- `variableName` - symbol name that is used for the lookup of the benchmark specification.

I believe this is a good first set of the parameters for the annotation. We could probably replace `moduleName` with `fileName`, or remove it altogether, but now, this feels like a detail.